test/__snapshots__/serialize.test.ts.snap

```diff
@@ -12380,6 +12380,1890 @@ function __f0() {
+
+function __f12(__0) {
+  return (function() {
+    with({ function_prewarm_1: __function_prewarm_1 }) {
+
+return function /*preWarm*/(prewarmContext) {
+                prewarmContext.getOrInit(function_prewarm_1.PrewarmClients.DYNAMO);
+            };
+
+    }
+  }).apply(undefined, undefined).apply(this, arguments);
+}
+
+function __f13(__0, __1) {
+  return (function() {
+    with({ function_prewarm_1: __function_prewarm_1 }) {
+
+return async (args, preWarmContext) => {
+                    const dynamo = preWarmContext.getOrInit(function_prewarm_1.PrewarmClients.DYNAMO);
+                    const [input] = args;
+                    const { Table: table, AttributesToGet, ...rest } = input;
+                    return dynamo
+                        .scan({
+                        ...rest,
+                        AttributesToGet: AttributesToGet,
+                        TableName: table.tableName,
+                    })
+                        .promise();
+                };
+
+    }
+  }).apply(undefined, undefined).apply(this, arguments);
+}
+
+function __f14(__0) {
+  return (function() {
+    with({ function_prewarm_1: __function_prewarm_1 }) {
+
+return function /*preWarm*/(prewarmContext) {
+                prewarmContext.getOrInit(function_prewarm_1.PrewarmClients.DYNAMO);
+            };
+
+    }
+  }).apply(undefined, undefined).apply(this, arguments);
+}
+
+function __f15(__0, __1) {
+  return (function() {
+    with({ function_prewarm_1: __function_prewarm_1 }) {
+
+return async ([request], preWarmContext) => {
+                    const eventBridge = preWarmContext.getOrInit(function_prewarm_1.PrewarmClients.EVENT_BRIDGE);
+                    return eventBridge
+                        .putEvents({
+                        Entries: request.Entries.map((e) => ({
+                            ...e,
+                        })),
+                    })
+                        .promise();
+                };
+
+    }
+  }).apply(undefined, undefined).apply(this, arguments);
+}
+
+function __f16(__0) {
+  return (function() {
+    with({ function_prewarm_1: __function_prewarm_1 }) {
+
+return (prewarmContext) => {
+                    prewarmContext.getOrInit(function_prewarm_1.PrewarmClients.EVENT_BRIDGE);
+                };
+
+    }
+  }).apply(undefined, undefined).apply(this, arguments);
+}
+
+function __f17() {
+  return (function() {
+    with({ m: __m, k: \\"$AWS\\" }) {
+
+return function () { return m[k]; };
+
+    }
+  }).apply(undefined, undefined).apply(this, arguments);
+}
+
+function __f0() {
+  return (function() {
+    with({ _fnls_1: __fnls_1, bus: __bus }) {
+
+return () => {
+                return _fnls_1.$AWS.EventBridge.putEvents({
+                    Entries: [
+                        {
+                            EventBusName: bus.eventBusName,
+                        },
+                    ],
+                });
+            };
+
+    }
+  }).apply(undefined, undefined).apply(this, arguments);
```

Comment:
Does this have any performance impact on V8's ability to optimize this code?

Reply:
No idea, its what the closure serializer generates and have not played with it yet. Not 100% sure they need to wrap all of the function like this.

Reply:
Yeah I know we inherited this. Thought I would ask the question anyway. I'm curious of the answer.

Reply:
yeah, same, slowly making changes.